README.mkd

```diff
@@ -229,6 +229,41 @@ registers and then save `ra` and the appropriate number of registers from
 epilogue. They restore the saved registers, deallocate the stack space for the
 register, and then perform a return through the restored value of `ra`.
 
+## Conventions for vendor extension
+
+Custom instruction is an important part of RISC-V, offical ISA spec has reserved
+serveral encoding space for extending instructions.
+
+However there is no offical rule on how to use those encoding space and the rule
```

Comment:
"However, there are no official guidelines on how to use this encoding space or on naming the mnemonics. This section defines guidelines which vendors are expected to follow if upstreaming support for their extensions. Although vendor-provided toolchains are free to make different choices, they are strongly urged to align with these guidelines in order to ensure there is a straightforward path for upstreaming in the future."

Reply:
I don't know if we want to talk about the 'encoding space' here, as we are not establishing any guidelines on how to use/subdivide/etc. the encoding space.

Reply:
Removed about encoding.